.gitignore

```diff
@@ -24,7 +24,7 @@ bin/
 # metals
 /.bloop/
 /.metals/
-/project/metals.sbt
+metals.sbt
```

Comment:
I would prefer that we be more strict. I've been using
```
/project/**/metals.sbt
```
in my projects.

Reply:
I didn't even know that `**` worked. Thanks.